snap/snapcraft.yaml

```diff
@@ -171,35 +165,38 @@ parts:
       - python3-rados
       - python3-rbd
       - sed
-      - socat
```

Comment:
socat was introduced to address the following error: https://github.com/canonical/microk8s/pull/94

could you please add some comment showing this is not needed anymore?

Reply:
`socat` is not needed for some time now https://github.com/containerd/cri/pull/1470:

```
root@u1:~# find /snap/microk8s/current | grep socat
root@u1:~# which socat
root@u1:~# k port-forward deploy/nginx 8080:80 &
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
root@u1:~# curl localhost:8080
Handling connection for 8080
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
html { color-scheme: light dark; }
body { width: 35em; margin: 0 auto;
font-family: Tahoma, Verdana, Arial, sans-serif; }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
```